src/plugins/discourse/createGraph.js

```diff
@@ -173,19 +199,26 @@ class _GraphCreator {
       this.graph.addEdge(authorsTopicEdge(serverUrl, topic));
     }
 
+    const postDescriptions: Map<PostId, string> = new Map();
     for (const post of data.posts()) {
-      this.addPost(post);
+      const description = this.addPost(post);
+      postDescriptions.set(post.id, description);
     }
 
     for (const like of data.likes()) {
+      const postDescription =
+        postDescriptions.get(like.postId) || "[unknown post]";
+      this.graph.addNode(likeNode(serverUrl, like, postDescription));
       this.graph.addEdge(likesEdge(serverUrl, like));
+      this.graph.addEdge(createsLikeEdge(serverUrl, like));
```

Comment:
Nit: inconsistent with other node types, which use an `addLike` style method.

Reply:
Refactored for consistency.